tools/generate_docker_image_names.py

```diff
@@ -0,0 +1,112 @@
+"""
+Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+バージョン文字列付きのDockerイメージ名を改行区切りで標準出力に出力する
+
+例1
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ",A,B"
+REPO:VER
+REPO:A-VER
+REPO:B-VER
+
+例2
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ""
+REPO:VER
+"""
+
+from argparse import ArgumentParser
+
+
+def generate_docker_image_names(
+    repository: str, version: str, comma_separated_prefix: str
+) -> list[str]:
+    """
+    Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+    バージョン文字列付きのDockerイメージ名を配列で返す
+
+    prefixが空文字列でない場合、"{prefix}-{version}"をタグにする
+
+    - 例: repository="REPO", version="VER", prefix="A" -> "REPO:A-VER"
+
+    prefixが空文字列の場合、"{version}"をタグにする
+
+    - 例: repository="REPO", version="VER",  prefix="" -> "REPO:VER"
+
+    Parameters
+    ----------
+    repository : str
+        Dockerリポジトリ名（例: "voicevox/voicevox_engine"）
+    version : str
+        バージョン文字列（例: "0.22.0", "latest"）
+    comma_separated_prefix : str
+        カンマ区切りのプレフィックス（例: ",cpu,cpu-ubuntu22.04", "nvidia,nvidia-ubuntu22.04"
+
+    Returns
+    -------
+    list[str]
+        Dockerイメージ名の配列
+        （例: [
+            "voicevox/voicevox_engine:0.22.0",
+            "voicevox/voicevox_engine:cpu-0.22.0",
+            "voicevox/voicevox_engine:cpu-ubuntu22.04-0.22.0"
+        ]）
+    """
+    # カンマ区切りのタグ名を配列に変換
+    prefixes = comma_separated_prefix.split(",")
+
+    # 戻り値の配列
+    docker_image_names: list[str] = []
+
+    for prefix in prefixes:
+        # プレフィックスが空文字列でない場合、末尾にハイフンを付ける
+        if prefix:
+            prefix = f"{prefix}-"
+        docker_image_names.append(f"{repository}:{prefix}{version}")
+
+    return docker_image_names
+
+
+def main() -> None:
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--repository",
+        type=str,
+        required=True,
+        help="Dockerリポジトリ名（例: voicevox/voicevox_engine）",
+    )
+    parser.add_argument(
+        "--version",
+        type=str,
+        default="latest",
+        help="バージョン文字列。デフォルトブランチの場合、latest",
+    )
+    parser.add_argument(
+        "--prefix",
+        type=str,
+        default="",
+        help="Dockerイメージのタグのカンマ区切りのプレフィックス",
+    )
```

Comment:
プログラムの引数のヘルプを関数の説明文に合わせます。

```suggestion
    parser.add_argument(
        "--repository",
        type=str,
        required=True,
        help="Dockerリポジトリ名（例: voicevox/voicevox_engine）",
    )
    parser.add_argument(
        "--version",
        type=str,
        default="latest",
        help="バージョン文字列（例: \"0.22.0\", \"latest\"）",
    )
    parser.add_argument(
        "--prefix",
        type=str,
        default="",
        help="カンマ区切りのプレフィックス（例: \",cpu,cpu-ubuntu22.04\", \"nvidia,nvidia-ubuntu22.04\"）",
    )
```